pkg/k8s/watchers/cilium_local_redirect_policy.go

```diff
@@ -72,7 +72,7 @@ func (k *K8sWatcher) ciliumLocalRedirectPolicyInit(ciliumLRPClient *k8s.K8sCiliu
 	)
 
 	go lrpController.Run(wait.NeverStop)
-	k.k8sAPIGroups.addAPI(k8sAPIGroupCiliumClusterwideNetworkPolicyV2)
+	k.k8sAPIGroups.addAPI(k8sAPIGroupCiliumLocalRedirectPolicyV2)
```

Comment:
Q: how did you test that an LRP svc doesn't get removed?

Reply:
Restarted cilium-agent a few times while LRP was still present, and monitored that this logs doesn't show up - "Deleting no longer present service"